docs/src/BrandingCssVarsProvider.tsx

```diff
@@ -1,6 +1,14 @@
 import * as React from 'react';
 import { deepmerge } from '@mui/utils';
-import { ThemeProvider, createTheme, PaletteColorOptions } from '@mui/material/styles';
+import {
+  Experimental_CssVarsProvider as CssVarsProvider,
+  // @ts-expect-error need to use deprecated API because MUI X repo still on Material UI v5
+  experimental_extendTheme as extendTheme,
+  createColorScheme,
```

Comment:
This does not seem to work. 🙈 
https://app.circleci.com/pipelines/github/mui/mui-x/82297/workflows/ffc58350-4471-4c39-a160-357f1cdd36da/jobs/468718?invite=true#step-109-2430_145
Looks like it's already time to bump `@mui/material` in docs. 😆

Reply:
I will find a workaround. It should be doable to avoid the type error.